Log/lambda_function.py

```diff
@@ -216,18 +216,11 @@ def is_cloudtrail(key):
     return bool(match)
 
 def parse_event_source(event, key):
-    if "lambda" in key:
-        return "lambda"
+    for source in ["lambda", "elasticloadbalancing", "redshift", "cloudfront", "kinesis", "mariadb", "mysql", "apigateway", "route53", "vpc", "rds"]:
+        if source in key:
+            return source
     if is_cloudtrail(str(key)):
         return "cloudtrail"
-    if "elasticloadbalancing" in key:
-        return "elb"
```

Comment:
Is this expected? Do we want to rename `elb` into `elasticloadbalancing`?

Reply:
Indeed, our integration name is elb but AWS put elasticloadbalancing in the file path